impl_core/src/main/java/io/opencensus/trace/export/SampledSpanStoreImpl.java

```diff
@@ -119,26 +128,32 @@ private Bucket getErrorBucket(CanonicalCode code) {
 
     private void considerForSampling(SpanImpl span) {
       Status status = span.getStatus();
-      Bucket bucket =
-          status.isOk()
-              ? getLatencyBucket(span.getLatencyNs())
-              : getErrorBucket(status.getCanonicalCode());
-      bucket.considerForSampling(span);
+      // Null status means running Span. Ignore this Span.
+      if (status != null) {
```

Comment:
You are correct, it is more to protect the library to not crash. Updated the comment.